Tasks/ServiceFabricUpdateAppVersions/Compare-Xml.psm1

```diff
@@ -0,0 +1,25 @@
+﻿function Compare-Xml
+{
+    [CmdletBinding()]
+    [OutputType([bool])]
+    Param
+    (
+        [Parameter(Mandatory=$true)]
+        [System.Xml.XmlNode]
+        $Item1,
+
+        [Parameter(Mandatory=$true)]
+        [System.Xml.XmlNode]
+        $Item2
+    )
+
+    Add-Type -AssemblyName System.Xml.Linq
+
+    $xmlReader1 = [System.Xml.XmlNodeReader] $Item1
+    $xmlReader2 = [System.Xml.XmlNodeReader] $Item2
+
+    $xDoc1 = [System.Xml.Linq.XDocument]::Load($xmlReader1)
+    $xDoc2 = [System.Xml.Linq.XDocument]::Load($xmlReader2)
+
+    [System.Xml.Linq.XNode]::DeepEquals($xDoc1, $xDoc2)
```

Comment:
Do you happen to know what version of .Net this requires? It makes me wonder whether the agent should recommend a minimum version of .Net during config time.